intermine/im-tables
# fixes tooltip issue #193

This fixes an issue where tooltips were appearing unstyled. After investigation this was a combination of two issues - one that we were missing a tiny bit of CSS after upgrading bootstrap, the other that jQuery UI's `tooltip()` method was overriding bootstrap's `tooltip()` method (argh). Both have been fixed. 

To test: 

- clone this pr
- build using `npm run dev`
- go to http://localhost:9000/test/test-load-from-global.html and confirm that tooltips for the column headers display es expected.
